data/events/2023-new-york-city.yml

```diff
@@ -0,0 +1,130 @@
+enddate: # The end date of your event. Leave blank if you don't have a venue reserved yet.
+
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start: 2023-03-10T00:00:00-05:00 # start accepting talk proposals.
+cfp_date_end: 2023-05-10T00:00:00-05:00 # close your call for proposals.
+cfp_date_announce: 2023-05-17T00:00:00-05:00 # inform proposers of status
+
+cfp_link: "https://www.papercall.io/dodnyc2023" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
+
+registration_date_start: 2023-04-01T00:00:00-05:00
+registration_date_end: 2023-06-04T00:00:00-05:00
+
+registration_closed: "" #set this to true if you need to manually close registration before your registration end date
+registration_link: "" # If you have a custom registration link, enter it here. This will control the Registration menu item as well as the "Register" button.
+
+# Location
+#
+coordinates: "" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+location: "" # Defaults to city, but you can make it the venue name.
+#
+location_address: "" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+  - name: propose
+  - name: location
+  - name: registration
+  - name: program
+  - name: speakers
+  - name: sponsor
+  - name: contact
+  - name: conduct
+# - name: example
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
+  - name: "Sumit Agarwal"
+    employer: "Broadridge Financial Solutions"
+    image: "sumit-agarwal.jpg"
+    twitter: "aga_sumit"
+    linkedin: "https://www.linkedin.com/in/agarwalsumitfsdevconsultant/"
+  - name: "Ceren Ercen"
+    employer: "Grubhub/Seamless"
+    image: "ceren-ercen.jpg"
+  - name: "Russ Kaehler"
+    employer: "Memorial Sloan Kettering Cancer Center"
+    image: "russ-kaehler.jpg"
+    linkedin: "https://www.linkedin.com/in/russ-kaehler-514187110"
+  - name: "Matt Kuritz"
+    employer: "The Farmer's Dog"
+    image: "matt-kuritz.jpg"
+    github: "kuritz"
+    twitter: "_kuritz"
+  - name: "Priyanka Rao"
+    image: "priyanka-rao.jpg"
+    linkedin: "https://www.linkedin.com/in/priyanka-rao"
+    twitter: "pri_posterous"
+  - name: "Adarsh Shah"
+    image: "adarsh-shah.jpg"
+    employer: "shahadarsh.com"
+    website: "http://shahadarsh.com"
+    twitter: "shahadarsh"
+    github: "shahadarsh"
+  - name: "Stephen Thomas"
+    employer: "iconectiv"
+    linkedin: "https://www.linkedin.com/in/steve-thomas-4746bb/"
+    image: "stephen-thomas.jpg"
+  - name: "Matthew Titmus"
```

Comment:
Have we reached out to Matt to see if he has any interest in re-joining for the stretch run if we move the sync?